src/cljs/harja/views/urakka/kulut.cljs

```diff
@@ -755,64 +758,82 @@
   [e! app]
   (komp/luo
     (komp/piirretty (fn [this]
-                      (e! (tiedot/->HaeUrakanLaskutJaTiedot (select-keys (-> @tila/yleiset :urakka) [:id :alkupvm :loppupvm])))))
+                      (e! (tiedot/->HaeUrakanLaskutJaTiedot (select-keys (-> @tila/yleiset :urakka) [:id :alkupvm :loppupvm])))
```

Comment:
Pitäiskö tämän funkkarin "HaeUrakanLaskutJaTiedot" nimi muuttaa, jos se ei enää hae laskuja?
Tuossa alempana käytetään siis funkkaria "HaeUrakanLaskut", josta näin päättelin

Reply:
Hyvä kommentti! Muutin tota ->HaeUrakanToimenpiteetJaMaksuerat